ga/latest/kernel/helpers/build/features-installed.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+if [ -f "/logs/configure-liberty.log" ]; then
+  rm /logs/configure-liberty.log
+  exit 0
+fi  
+  
+>&2 echo "WARNING: This is not an optimal build configuration. Although features in server.xml will continue to be installed correctly, the 'RUN features.sh' command should be added to the Dockerfile prior to configure.sh. See https://github.com/WASdev/ci.docker#building-an-application-image for a sample application image template."
```

Comment:
use this short URL instead. If needed, we can update it to point elsewhere in the future: `ibm.biz/wl-app-image-template`

Reply:
Updated all the links to use the short URL